samples/snippets/csharp/VS_Snippets_CLR/CryptoWalkThru/cs/Form1.cs

```diff
@@ -131,19 +131,16 @@ private void buttonDecryptFile_Click(object sender, EventArgs e)
         private void EncryptFile(string inFile)
         {
 
-            // Create instance of Rijndael for
-            // symetric encryption of the data.
-            RijndaelManaged rjndl = new RijndaelManaged();
-            rjndl.KeySize = 256;
-            rjndl.BlockSize = 256;
-            rjndl.Mode = CipherMode.CBC;
-            ICryptoTransform transform = rjndl.CreateEncryptor();
+            // Create instance of Aes for
+            // symmetric encryption of the data.
+            Aes aes = Aes.Create();
+            ICryptoTransform transform = aes.CreateEncryptor();
 
             // Use RSACryptoServiceProvider to
-            // enrypt the Rijndael key.
+            // encrypt the AES key.
             // rsa is previously instantiated:
             //    rsa = new RSACryptoServiceProvider(cspp);
```

Comment:
Is the platform interop of RSACryptoServiceProvider here necessary to the sample, or can it be rewritten in terms of the RSA base class?  (If the interop is necessary to meet the associated prose that's fine)

Reply:
>Is the platform interop of RSACryptoServiceProvider here necessary to the sample, or can it be rewritten in terms of the RSA base class? (If the interop is necessary to meet the associated prose that's fine)

The text mentions the key container name and the form includes Labels for the key container name and code that uses CspParameters.KeyContainerName.